example/krakend.json

```diff
@@ -7,21 +7,45 @@
     "endpoints": [
         {
             "endpoint": "/github/{user}",
+			"method": "GET",
+			"output_encoding": "no-op",
             "backend": [
                 {
                     "host": [
-                        "github"
+                        "github.com"
```

Comment:
we defined a "non compliant service name" in order to make it clear that these should be etcd keys